README.md

```diff
@@ -503,7 +503,7 @@ Once you're done, add your parser to the [zones.json](https://github.com/tmrowco
 
 Run all of the parser tests with the following command from the root directory:
 ```
-python3 -m unittest discover parsers/test/
```

Comment:
I found it helpful to know that these were python3 when getting things up and running, are you sure it's a good idea to remove this?

Reply:
From my perspective this is system dependant. I, for example, don't have a 'python3' command - I use conda environments.

And in the other command line snippets in the readme, we use 'python', so I think this is better for consistency's sake...

But we should definitely specify somewhere that we use python 3.6.. I will add this to the readme now